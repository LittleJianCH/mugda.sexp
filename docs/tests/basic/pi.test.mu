(Pi ((A Type)) (-> A A))
