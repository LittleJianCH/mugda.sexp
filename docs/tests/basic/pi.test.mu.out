(Pi Type (closure A (Pi A (closure _ A))))
